internal/kafka/franz_client.go

```diff
@@ -318,6 +319,19 @@ func configureKgoSASL(cfg *configkafka.SASLConfig) (kgo.Opt, error) {
 			token, _, err := signer.GenerateAuthToken(ctx, cfg.AWSMSK.Region)
 			return oauth.Auth{Token: token}, err
 		})
+	case OIDCFILE:
+		m = oauth.Oauth(func(ctx context.Context) (oauth.Auth, error) {
+			tokenProvider, cancel := NewOIDCTokenProvider(ctx, clientID,
+				cfg.OIDCFILE.ClientSecretFilePath, cfg.OIDCFILE.TokenURL,
+				cfg.OIDCFILE.Scopes, cfg.OIDCFILE.EndPointParams,
+				cfg.OIDCFILE.AuthStyle, cfg.OIDCFILE.ExpiryBuffer)
+			_ = cancel // Store cancel function for cleanup if needed
```

Comment:
This is just assigning the cancel function to the blank operator, did you mean to do something more with it?

Reply:
Good catch - I originally thought perhaps it might be needed/useful for a context cancellation invocation, but that's not done anywhere in this call-chain on upward - removed in commit f1d88f04ae .